src/PostTypes/Course.php

```diff
@@ -529,6 +523,8 @@ protected function createPostType()
                  'menu_icon'          => 'dashicons-shield-alt',
              );
 
+            $args = apply_filters('admwpp_course_args', $args);
```

Comment:
sorry for this @alihabib-administrate can we change the name of the filter to be generic.
`admwpp_" .  self::$slug . "_args`

Reply:
in case we changed the course slug this will also change the filter name handler.

Reply:
but this is in the course.php class